jni/include/faiss_wrapper.h

```diff
@@ -47,11 +47,21 @@ namespace knn_jni {
         // Return a pointer to the loaded index
         jlong LoadIndex(knn_jni::JNIUtilInterface * jniUtil, JNIEnv * env, jstring indexPathJ);
 
+        // Loads an index with a reader implemented IOReader
+        //
+        // Returns a pointer of the loaded index
+        jlong LoadIndexWithStream(faiss::IOReader* ioReader);
```

Comment:
Lets move these methods to index service?

Reply:
This is a parity of `LoadIndex` which will be deprecated soon.
Tried not to modify the whole structure too much.

Reply:
Load index is a pending refactor/move, if there is enough bandwidth we should move to index service.

@heemin32 thoughts?

Reply:
Can I make the changes in the upcoming PR?
Which will be raised right after this - Streaming support NMSLIB.
Will ask you for the review for that.

Reply:
Better to have it in index service. Having it in the next PR is fine for me.

Reply:
Sounds good! 
Will move the method in index service in the next PR where I will also ask you to review over the refactoring.
Thank you both.